openfisca_us/parameters/gov/states/il/tax/income/personal_exemption/dependent_exemption.yaml

```diff
@@ -0,0 +1,11 @@
+description: Illinois Dependent Exemption Amount
```

Comment:
rename to `dependent` (same for `personal`, `aged_blind`)

Reply:
[rename of exemption parameter structure](https://github.com/PolicyEngine/openfisca-us/pull/1056/commits/3a4a3fad34622fa880305934ee7eb451e35c2956)